FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentLobby.cs

```diff
@@ -46,15 +50,19 @@ public unsafe partial struct LobbyData {
 
     [FieldOffset(0x858)] public StdVector<Pointer<CharaSelectCharacterEntry>> CharaSelectEntries;
 
+    [MemberFunction("40 53 56 41 57 48 83 EC 20 33 DB")]
+    public partial CharaSelectCharacterEntry* GetCharacterEntryFromServer(byte index, long contentId);
+
     [MemberFunction("E8 ?? ?? ?? ?? 48 85 C0 74 95")]
     public partial CharaSelectCharacterEntry* GetCharacterEntryByIndex(int a2, int worldIndex, int characterMappingIndex);
 }
 
 [VTableAddress("48 8D 05 ?? ?? ?? ?? 48 8B F9 48 89 01 48 81 C1 ?? ?? ?? ?? E8 ?? ?? ?? ?? 48 8D 8F ?? ?? ?? ?? E8 ?? ?? ?? ?? 48 8D 8F ?? ?? ?? ?? E8 ?? ?? ?? ?? 48 8D 8F", 3)]
 [StructLayout(LayoutKind.Explicit, Size = 0x848)]
 public unsafe partial struct LobbyUIClient {
-    //[FieldOffset(0x10)] public NetworkModuleProxy* NetworkModuleProxy;
-    //[FieldOffset(0x18)] public ?* SomeNetworkConfig; // contains hosts and ports
+    [FieldOffset(0x00)] public void** vtbl;
+    [FieldOffset(0x10)] public NetworkModuleProxy* NetworkModuleProxy;
+    //[FieldOffset(0x18)] public ?* NetworkModule; // contains hosts and ports
```

Comment:
Does this point to the `NetworkModule` struct you added?

```suggestion
    [FieldOffset(0x18)] public NetworkModule* NetworkModule;
```

Reply:
No, unfortunately not. Thought this at first

Reply:
Oh, ok. 🙂
I tried to mark it as resolved, but it only hid my comment, lol.

While we're here, you might want to look at the `ClientTime` struct you added. It's of size 0x40 and inherits a 0x20 `Common::Game::Time::GameTime` struct (not so important, just wanted to note). The reason I'm saying this is because `Framework` has fields that are part of `ClientTime` and without the size it's small enough for CExporter to not complain.

Reply:
The `Time` field is updated at `48 C1 E8 09 48 89 87`. It's the `long EorzeaTime` field in `Framework`.